rust/src/cliwrap/yumdnf.rs

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use anyhow::{anyhow, Result};
+use indoc::formatdoc;
+use indoc::indoc;
+use std::os::unix::process::CommandExt;
+use std::path::Path;
+use std::process::Command;
+use structopt::StructOpt;
+
+/// Emitted at the first line.
+const IMAGEBASED: &str = "Note: This system is image (rpm-ostree) based.";
+
+/// Emitted for unhandled options.
+const UNHANDLED: &str = indoc::indoc! {r#"
+rpm-ostree: Unknown arguments passed to yum/dnf compatibility wrapper.
+
+Please see `yum --help` for currently accepted commands via this
+wrapper, and `rpm-ostree --help` for more information about.the rpm-ostree
+project.
+"#};
+
+const OTHER_OPTIONS: &[(&str, &str)] = &[
+    (
+        "toolbox",
+        "For command-line development and debugging tools in a privileged container",
+    ),
+    ("podman", "General purpose containers"),
```

Comment:
Hm, perhaps these should be RPM names and then we can output `rpm -q --qf %{summary}` text instead.